go/pkg/cas/client.go

```diff
@@ -147,6 +164,15 @@ func DefaultClientConfig() ClientConfig {
 			MaxSizeBytes: 4 * 1024 * 1024,
 			Timeout:      time.Minute,
 		},
+		ByteStreamWrite: RPCConfig{
+			Concurrency: 256,
+			// 4MiB is the default gRPC request size limit.
+			MaxSizeBytes: 4 * 1024 * 1024,
+			Timeout:      time.Minute,
```

Comment:
note: the comment above says that these fields apply to each request in a stream (i.e. not entire stream). From that perspective, 1m per chunk is more than enough (The point of timeout is to prevent hanging, but if we are making progress - so I think it should be per request, not stream).

Reply:
Ah, I missed your TODO comment regarding per-RPC timeouts - which means so far this Timeout is being ignored? sgtm